### PR TITLE
CreateImageWizard: update review page

### PR DESCRIPTION
**Pull request by jkozol**
The review page now uses more patternfly components and has been split up a little so it is easier to add review fields for additional target environments.

### CODE REVIEW

**Comment by jkozol** [comment]
Switching to draft as I make changes

**Comment by jkozol** [comment]
review

File: src/PresentationalComponents/CreateImageWizard/WizardStepReview.scss [hunk 1]
```diff
@@ -14,4 +13,9 @@
         grid-template: 1fr / 25% 1fr;
         grid-gap: var(--pf-global--spacer--sm);
     }
+}
+
+#destination-header {
+    font-size: 18px;
+    margin-bottom: var(--pf-global--spacer--sm);
 }
```
Comment (Gundersanne) [inline src/PresentationalComponents/CreateImageWizard/WizardStepReview.scss:21]: nitpick, but trailing newline please

**Review by Gundersanne — COMMENTED** [review]
1 nitpick, and 1 question:

Is there an advantage over using the patternfly `Text*` components vs the regular `p/dt/dl/etc...` elements? Like do the pf components have extra styling applied? I'm fine with it  either way I just wonder what the difference is.

File: src/PresentationalComponents/CreateImageWizard/WizardStepReview.scss (hunk 1 above)
Comment (jkozol) [reply src/PresentationalComponents/CreateImageWizard/WizardStepReview.scss:21]: hehe i'm so bad at this one. Will fix.

**Comment by jkozol** [comment]
> Is there an advantage over using the patternfly Text* components vs the regular p/dt/dl/etc... elements? Like do the pf components have extra styling applied? I'm fine with it either way I just wonder what the difference is.

@Gundersanne I am not completely sure. I have just seen plenty of their examples use it. If i had to guess it is because patternfly uses a few css variables over many components so using their Text* components will make sure all of our styles stay fairly consistent. @kkoukiou do you have a better answer for this?

**Review by Gundersanne — APPROVED** [review]
lgtm! thanks

**Pull request merged**